### DISCUSSION TITLE
ERROR: Tree file does not start with an opening-bracket '(' (line 1 column 1)

### DISCUSSION THREAD

**Discussion by srgirard** [iqtree/iqtree2#178] [Q&A]
Hello, I am trying to get bootstraps, gCF, and sCF on a concatenated/supermatrix tree using the command: 

iqtree2 -t <supermatrix_tree> --gcf <concat_loci> -p <align_directory> --scf 100 --prefix final -B 1000 -T 4

This is running for a few hours before failing with a ERROR: Tree file does not start with an opening-bracket '(' (line 1 column 1)

Does anyone know what is causing this?

Thanks!!

**Comment by roblanf** [comment]
@srgirard it's hard to help without more information. If you are able to post your input files and log file, that would help a lot.

In the absence of that information - have you double checked the treefile? Does the error message help? It's very specific.

**Reply by srgirard** [reply]
Hi, thanks for the reply. 

I looked closer into the log file and I realized that I used the wrong file as the --gcf input file. But now that I have the right file I am still having issues because all of the trees in my loci.treefile file have less tips than my concatenated supermatrix tree made from the same alignment files. 

I used this for the reference tree
iqtree2 -p <alignment_directory> --prefix concat -B 1000 -T AUTO

To make the loci.tree file I used 
iqtree2 -S <alignment_directory> --prefix loci -T AUTO

And I used the same command as the original post (but now with the right -- gcf input file) and am getting the error 
ERROR: Tree taxa and alignment sequence do not match 
Do you know why they would have a different number of tips? It appears that all the trees in the loci.treefile have between 87-107 tips even though my input taxa and reference tree both are at 111. 

Thanks

Here's the last 50 lines or so from the log file
[bs_gcf_scf_log.txt](https://github.com/iqtree/iqtree2/files/15108955/bs_gcf_scf_log.txt)

**Comment by thomaskf** [comment]
Hi srgirard,

Can you try the latest version like 2.3.x ? The issue you encountered
appeared in the previous version but it should have been fixed in the
latest version.

Thanks,
Thomas

**Reply by srgirard** [reply]
Hello, thanks for the reply. When I first tried I was using 2.3.0, and I have updated to 2.3.3 and I am still getting the same exact error, with he same exact individuals shown in the new log. The log I posted and the new log are identical. I tried making a brand new supermatrix tree and a new loci.tree file but still the same error.

**Comment by thomaskf** [comment]
Hi Srgirard,
Thanks for your question.

Can I confirm whether you used the latest version to run the following
commands to make the loci tree file again?

iqtree2 -S <alignment_directory> --prefix loci -T AUTO

If so, then can you send us the data files, perhaps by email if more
convenient for you, and the whole (not partial) log file? We will then have
a check.

Thanks,

Thomas

**Reply by srgirard** [reply]
Hello, yes! I just reran it using v2.3.3 and I am getting the same missing tips errors as when I was using an older 2.2.x version. What is the email I could get the log file and input data files to you?

**Comment by thomaskf** [comment]
Hi,
Got it. You can send the data files to me at ***@***.***
I will check and let you know. Thanks.
Thomas

**Comment by thomaskf** [comment]
Hi @srgirard 

Sorry for the delay of the reply.
These were the commands you used:
a.	Build a loci tree for each locus alignment:
`iqtree2 -S align_directory --prefix loci -nt 20`
b.	Build a tree for the concatenate alignment:
`iqtree2 -p align_directory --prefix concat -B 1000 -T 20`
c.	Perform gcf and scf analysis
`iqtree2 -t concat.treefile --gcf loci.treefile -p align_directory --scf 100 --prefix final -B 1000 -nt 20`

However, for step c, you should not include "-B 1000", because you have already did that in step b.
If you change the command in step c as follows, then it should complete successfully:
`iqtree2 -t concat.treefile --gcf loci.treefile -p align_directory --scf 100 --prefix final -nt 20`

Thanks for @bqminh 's help in coming up with the above answer.

Thomas
